public_suffix_list.dat

```diff
@@ -15596,6 +15596,8 @@ web.val.run
 vercel.app
 vercel.dev
 now.sh
+vusercontent.net
+v0.build
```

Comment:
Just a reminder to sort your entries according to the [guidelines](https://github.com/publicsuffix/list/wiki/Guidelines#sort-your-submission-correctly-important). Keep the list of suffixes (if there is more than one) in alphabetical order. Sort first by the TLD, then by the first label to the left of the TLD, and so on.